docs/README.md

```diff
@@ -165,7 +165,9 @@ Info: kruize deploy succeeded: Running
 kruize-5cd5967d97-tz2cb                        1/1     Running   0          24s
 ```
 
-Now you need to install the Kruize Dashboard, see the [Install the Grafana dashboard](#install-the-grafana-dashboard) section for more details.
+Now you need to install the Kruize Dashboard, see the [Install the Grafana dashboard](#install-the-grafana-dashboard) section for more details. 
+
+Note: In some OpenShift setups adding a custom dashboard is not possible, and in those cases visualization through the Kruize dashboard is not currently supported.
```

Comment:
Please be more specific and add OpenShift version numbers. OpenShift versions <=3 do not support adding grafana dashboards. Version 4.4 onwards we have a separate Prometheus / Grafana instance that can be deployed that can be used to monitor applications. (This is currently tech preview only)